### ISSUE TITLE
Test mechanism is creating too many node processes

### ISSUE THREAD

**Issue by ajsantander** [aragon/aragon-cli#1083]
After running tests several times, running pgrep shows that several nodejs processes remain running.

**Comment by dapplion** [comment]
Maybe this be related to the over-use of subprocesses? Or is it due to spinning IPFS and Ganache in the background?

**Comment by 0x6431346e** [comment]
```sh
aragon-cli on  fix/ipfs-view via ⬢ v10.18.1 
➜ pgrep node
aragon-cli on  fix/ipfs-view via ⬢ v10.18.1 
➜ npm run pretest 

> @aragon/cli-monorepo@ pretest /home/daniel/repos/aragon/aragon-cli
> node scripts/setup-integration-tests.js

[cli-monorepo:pretest] Project already initialized: ./.tmp/setup/ipfs-project
[cli-monorepo:pretest] Go-ipfs already installed: ./.tmp/setup/ipfs-project
[cli-monorepo:pretest] Devchain starting on: 8545
[cli-monorepo:pretest] Daemon starting on port: 5001
[cli-monorepo:pretest] spawned subprocess with PID:  2495
[cli-monorepo:pretest] stdout: Initializing daemon...
go-ipfs version: 0.4.22-
Repo version: 7
System version: amd64/linux
Golang version: go1.12.7

[cli-monorepo:pretest] stdout: Swarm listening on /ip4/127.0.0.1/tcp/4001
Swarm listening on /ip4/192.168.1.34/tcp/4001
Swarm listening on /ip6/::1/tcp/4001
Swarm listening on /p2p-circuit

[cli-monorepo:pretest] stdout: Swarm announcing /ip4/127.0.0.1/tcp/4001
Swarm announcing /ip4/192.168.1.34/tcp/4001
Swarm announcing /ip6/::1/tcp/4001
API server listening on /ip4/0.0.0.0/tcp/5001
WebUI: http://0.0.0.0:5001/webui
Gateway (readonly) server listening on /ip4/0.0.0.0/tcp/8080
Daemon is ready

[cli-monorepo:pretest] Daemon started on port: 5001
[cli-monorepo:pretest] Daemon ready!!

[cli-monorepo:pretest] Devchain started on: 8545
[cli-monorepo:pretest] Devchain ready!!
aragon-cli on  fix/ipfs-view via ⬢ v10.18.1 took 3s 
➜ pgrep node
2488
aragon-cli on  fix/ipfs-view via ⬢ v10.18.1 
➜ npm run posttest

> @aragon/cli-monorepo@ posttest /home/daniel/repos/aragon/aragon-cli
> node scripts/teardown-integration-tests.js

[cli-monorepo:posttest] Processes killed on ports: 5001, 8545
aragon-cli on  fix/ipfs-view via ⬢ v10.18.1 
➜ pgrep node
aragon-cli on  fix/ipfs-view via ⬢ v10.18.1
➜ 
```

Seems to work fine on my machine, can you provide some logs/more info @ajsantander ?
